docs/articles/getting-started.md

```diff
@@ -95,52 +96,52 @@ private async void button1_Click(object sender, EventArgs e)
 
-The following Devices are available as properties on the [`IChroma`](xref:Colore.IChroma) instance:
+The following Devices are available as properties on the `IChroma` instance:
 
- * [`Mousepad`](xref:Colore.IMousepad)
- * [`Keypad`](xref:Colore.IKeypad)
- * [`Headset`](xref:Colore.IHeadset)
- * [`Keyboard`](xref:Colore.IKeyboard)
- * [`Mouse`](xref:Colore.IMouse)
+ * `Mousepad`
+ * `Keypad`
+ * `Headset`
+ * `Keyboard`
+ * `Mouse`
 
 As a mousepad doesn't contain a SetKey method take a look at section 5.
 
-## 4. How to define custom Colors
-You can of course define any color you like when you use [`new ColoreColor(1.0, 1.0, 1.0)`](xref:Colore.Data.Color#Colore_Data_Color__ctor_System_Double_System_Double_System_Double_). The numbers are in the order red, green, and blue where `1.0` is full color and `0.0` is nothing. Which makes this white (red, green and blue glowing at full bright). You can also use color values which range from 0 - 255 (the usual color range) but make sure to use or cast your value to `byte` to call the correct constructor. This can cause some trouble if you aren't careful:
+## 5. How to define custom Colors
+You can of course define any color you like when you use `new ColoreColor(1.0, 1.0, 1.0)`. The numbers are in the order Red, Green and Blue where 1.0 is full color and 0.0 is nothing. Which makes this White (Red Green and Blue glowing at full bright). You can also use color values which range from 0 - 255 (the usual color range) but make sure to use or cast your value to `byte` to call the correct constructor. This can cause some trouble if you aren't careful:
 
-This will work as expected as it's using the byte constructor: [`new ColoreColor(255, 125, 125)`](xref:Colore.Data.Color#Colore_Data_Color__ctor_System_Byte_System_Byte_System_Byte_)
-This will **NOT** work as expected: [`new ColoreColor(255.0, 125.0, 125.0)`](xref:Colore.Data.Color#Colore_Data_Color__ctor_System_Double_System_Double_System_Double_)
+This will work as expected as it's using the byte constructor: `new ColoreColor(255, 125, 125)`
+This will **NOT** work as expected: `new ColoreColor(255.0, 125.0, 125.0)`
 
 This does work as expected:
 
 ``` C#
 int i = 255;
 var color = new ColoreColor((byte)i, (byte)i, (byte)i);
 ```
-Result: Color with R=255, G=255, B=255.
+Result: Color with R=255, G=255, B=255
 
 While this does **NOT** work as expected (the integers are coerced into floating point numbers):
 
 ``` C#
 int i = 255;
 var color = new ColoreColor(i, i, i);
 ```
-Result: Color with R=1, G=1, B=1.
+Result: Color with R=1, G=1, B=1
 
 
-## 5. How do I know which color is currently set on a Key / What if I don't want to set a Key but a specific row/column?
-Internally Colore is storing colors in a grid.
+## 6. How do I know which color is currently set on a Key / What if I don't want to set a Key but a specific row/column?
+Internally Colore is storing colors in a Grid.
```

Comment:
Here, "grid" is used as the actual word, so should not be capitalized. Same goes for the "grid" below.